lib/db_connection/sojourn.ex

```diff
@@ -35,9 +35,8 @@ defmodule DBConnection.Sojourn do
 
   @behaviour  DBConnection.Pool
 
-  @broker    DBConnection.Sojourn.Broker
-  @regulator DBConnection.Sojourn.Regulator
-  @timeout   15_000
+  @broker  DBConnection.Sojourn.Broker
```

Comment:
I think we should stop making values aligned, it breaks changes history, and makes unnecessary noise in PRs. Looks like `@behaviour` definition is already "broken" because of this reason, it has 2 spaces. :bowtie: